db/migrate/20211209153121_create_join_table_people_rooms.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTablePeopleRooms < ActiveRecord::Migration[6.1]
+  def change
+    create_join_table :people, :rooms do |t|
+      # t.index [:person_id, :room_id]
+      # t.index [:room_id, :person_id]
```

Comment:
why is this loop empty? i think these lines need to be added?

Reply:
I don't know, this code is auto-generated